sprint2/style.css

```
@@ -0,0 +1,22 @@
+body {
+    text-align: left;
+    padding-left: 50px;
+    border-top: 10px solid;
+    border-top-color: #005889;
+    font-size:20px;
+}
+.Contact {
```

```suggestion
.contact {
```

numele claselor le scrim cu litere mici in css